CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Contribution Guidelines
+
+Any change to this repository is always welcomed, and greatly appreciated. You can add new entries, or modify/remove existing entries if they are changed/discontinued. You are also allowed to add/modify categories as you see fit.
+
+Please ensure your pull request adheres to the following guidelines:
+
+-   Search previous suggestions before making a new one, as yours may be a duplicate.
```

Comment:
This may be a non-issue, the `duplicate` label exists for a reason?

Reply:
I think it is still useful to specify that one should check for duplicates before submitting a PR, so we don't have to check it ourselves. Although it does seem a bit redundant

Reply:
ok, that sounds fine